lib/ui/text.dart

```diff
@@ -568,28 +569,32 @@ Int32List _encodeParagraphStyle(
     result[0] |= 1 << 5;
     result[5] = maxLines;
   }
-  if (fontFamily != null) {
+  if (widthType != null) {
     result[0] |= 1 << 6;
+    result[6] = widthType.index;
```

Comment:
Since this is a binary value, it is possible to compress this into much smaller than the 32 bit int you are using now, but for now, this is OK. The benefit of what you have is that it is consistent and more readable. A more compressed version would be harder to read and maintain.

Reply:
It looks like I'm going to remove `widthType`, so this won't be a problem anymore.